plugins/inputs/jolokia/jolokia.go

```diff
@@ -14,7 +14,16 @@ import (
 	"github.com/influxdata/telegraf/plugins/inputs"
 )
 
-type Server struct {
+type JolokiaServer struct {
+	Name     string
+	Host     string
+	Username string
+	Password string
+	Port     string
+	Https    bool
+}
```

Comment:
Not sure I follow what the proxy has to do with it. But the proxy should be a URL too. All the code does is concatenate all the config params together and then parse it as a url.